src/App.tsx

```diff
@@ -1,8 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { getTodos, getUsers } from './api/todos&users/todos&users';
+import { TodoList } from './components/TodoList';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: React.FC = () => {
+  const [todos, setTodos] = useState<PreparedTodo[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
примитивы не указываем
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```